Tribler/Core/defaults.py

```diff
@@ -147,6 +148,11 @@
 sessdefaults['watch_folder']['enabled'] = False
 sessdefaults['watch_folder']['watch_folder_dir'] = None
 
+# HTTP API config
+sessdefaults['http_api'] = OrderedDict()
+sessdefaults['http_api']['enabled'] = True
```

Comment:
Sorry for noticing now, but I don't think we want this enabled by default yet...